repo.go

```diff
@@ -111,6 +113,8 @@ func (r *Repo) Init(consistentSnapshot bool) error {
 	}
 	root := data.NewRoot()
 	root.ConsistentSnapshot = consistentSnapshot
+	// Start a new root at version 1.
+	root.Version++
```

Comment:
nit: Should we just modify `data.NewRoot` to set `Version: 1`?

Reply:
That would be nice... The one bad part about doing it this way is that if you don't call `repo.Init` and just go from `NewRepo` to `AddKey`, then you'll accidentally increment twice, first from getting `NewRoot` and then since you haven't written to local yet, incrementing once more.

Reply:
Ah I see. Thanks for explaining